cabal-testsuite/PackageTests/NewConfigure/NewConfigure.cabal

```diff
@@ -0,0 +1,12 @@
+name:                NewConfigure
```

Comment:
Could you move this test to a subdirectory of `NewConfigure`? There may be other tests for `new-configure` we will want to add, and if you put it in `NewConfigure` directly you've used up the entire namespace!